otbox.py

```diff
@@ -206,38 +251,46 @@ def _excecute_command_safely(self, deviceType, deviceId, payload, cmd):
 
     def _mqtt_handler_echo(self, deviceType, deviceId, payload):
         '''
-        opentestbed/deviceType/box/deviceId/box1/cmd/echo
+        {0}/deviceType/box/deviceId/box1/cmd/echo
```

Comment:
I don't understand `{0}` in the comment. Is this a typo?

Applies throughout.

Reply:
It is not a typo, it is a "placeholder". Since the first MQTT topic level is now formed dynamically depending on the chosen testbed, I reckoned that the comment lines within the handlers should reflect that as well

Reply:
I see but it is a bit confusing. Is there some syntax we could use to denote this clearly, e.g. `TESTBED` in capital letters or something simila